src/main/java/org/mskcc/cmo/metadb/service/ValidRequestChecker.java

```diff
@@ -3,8 +3,15 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonMappingException;
 import java.io.IOException;
+import java.util.Map;
 
 public interface ValidRequestChecker {
     public String getFilteredValidRequestJson(String requestJson)
```

Comment:
also something that can be done outside of this PR - make the method declaration consistent (remove `public` from these interface methods)